packages/p2p/src/pages/my-ads/edit-ad-form.jsx

```diff
@@ -134,351 +134,304 @@ const EditAdForm = () => {
 
     return (
```

Comment:
let's add a sanity check in updating an ad to QA checklist

Reply:
was this page fine when you checked it @nada-deriv

Reply:
yeah.. the value for loading is never set to true